x/oracle/keeper/native_token.go

```diff
@@ -20,27 +20,28 @@ import (
 // undelegate: update operator's price, operator's totalAmount, operator's totalShare, staker's share
 // msg(refund or slash on beaconChain): update staker's price, operator's price
 
-type NSTETHAssetID string
+type NSTAssetID string
 
```

Comment:
some spelling erros in this file.

1. "deopsit" should be "deposit" (in the UpdateNSTValidatorListForStaker)  
2. "shoule" should be "should" (in the UpdateNSTByBalanceChange)  
3. "ingore" should be "ignore" (in the UpdateNSTByBalanceChange)  
4. "lenft" should be "left" (in the parseBalanceChangeCapped)